git-helper.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+action=$1
+option=$2
+
+if [[ "${action}" == "push" ]]; then
+  if [[ "${option}" == "test" ]]; then
+    git push staging master --no-verify
+    git push staging-mainnet master --no-verify
+
+    read -p "Do you wish to also perform a DB migrate to staging/staging-mainnet? (y/n) " yn
+    case $yn in
+        [Yy]* ) heroku run rake db:migrate -a set-core-staging; heroku run rake db:migrate -a set-core-staging-mainnet; break;;
+        [Nn]* ) echo "Skipping.";;
+        * ) echo "Skipping.";;
+    esac
+  elif [[ "${option}" == "prod" ]]; then
```

Comment:
We should avoid abbreviations whenever possible to maximize readability and search-ability of the codebase.